TextEditor/ModificationLogs/DeletionLog.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TextEditor
+{
+    class DeletionLog : IModification
+    {
+        private readonly char deletedSymbol;
+        public DeletionLog(char delSymb)
+        {
+            deletedSymbol = delSymb;
+        }
+
+        public void UndoModification()
+        {
+            TextEditor.Output += deletedSymbol;
```

Comment:
Using public static properties, and especially modifying them is a very bad practice. You can avoid it by injecting an instance of TextEditor into DeletionLog (as a constructor parameter).